### DISCUSSION TITLE
Not able to run release build

### DISCUSSION THREAD

**Discussion by har2008preet** [callstack/repack#811] [Q&A]
Hi,

I can create and run my host application which is using modules via ScriptManager, but when i create release build, it is crashing on start with error ReferenceError: Property '__webpack_require__' doesn't exist. I have already included export BUNDLE_COMMAND=webpack-bundle, but i think it is using metro to create release bundle. 

How can i confirm it what it is using to create bundle and how to resolve it?

**Comment by jbroma** [comment]
Hi,

`BUNDLE_COMMAND=webpack-bundle` should be enough, in newer versions of RN (>74) we override the `start` and `bundle` commands by default, so even that change should be optional.

what is the flow you're using to create the release build? Ideally please create a reproduction repository and detail the steps you're doing so we can resolve this, thanks!
